packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -1372,4 +1373,10 @@ export default class ScheduledPaymentModule {
       gasRangeInUSD,
     };
   }
+
+  async getValidForDays(): Promise<number> {
+    let configModuleAddress = await getAddress('usdStableCoinToken', this.ethersProvider);
```

Comment:
It is supposed to be `let configModuleAddress = await getAddress('scheduledPaymentConfig', this.ethersProvider);`.